blog/modules/ROOT/examples/sort-components/componentSorter-silly.js

```diff
@@ -0,0 +1,3 @@
+'use strict'
+
+module.exports = (components) => Object.values(components).reverse()
```

Comment:
Since this examples sorts the component list in descending order, wouldn't it be better to name the file `componentSorter-descending.js`? `silly` is an opinion, and doesn't convey what the example does without inspection.